apps/src/lib/node/ledger/events.rs

```diff
@@ -32,6 +35,7 @@ impl Display for EventType {
         match self {
             EventType::Accepted => write!(f, "accepted"),
             EventType::Applied => write!(f, "applied"),
+            EventType::Ibc(t) => write!(f, "{}", t),
```

Comment:
I think we should prefix the IBC events with e.g.:
```suggestion
            EventType::Ibc(t) => write!(f, "ibc:{}", t),
```
because Tendermint merges all the events in a block into one

(same on line 50)

Reply:
I'm not sure how Tendermint merges them. Do the event type strings are merged?
The event type string is parsed to convert a Tendermint event to `IbcEvent` in ibc-rs.
for example, https://github.com/informalsystems/ibc-rs/blob/ae364f9e93d055e63f49ac457f0acf6df110c62c/modules/src/core/ics02_client/events.rs#L31
from_str(): https://github.com/informalsystems/ibc-rs/blob/ae364f9e93d055e63f49ac457f0acf6df110c62c/modules/src/events.rs#L178

Reply:
as discussed the IBC relayer expects this format, so let's keep it as is